component/Review/ReviewAssignessModal.jsx

```diff
@@ -101,7 +101,7 @@ const GetAssigneesList = ({ type, reviewAssignee }) => {
             </div>
 
             <div className="flex-1">
-              <p className="mb-0 text-primary font-medium text-base">
+              <p className="mb-0  font-medium text-base">
                 {assignee?.assigned_to?.first_name}
```

Comment:
when sure  that there will always be assignee dont use .? operator

Reply:
okay